### PR TITLE
Cross DC query in Consul

### PR DESCRIPTION
**Pull request by guymarom**
Added an endpoint that filters healthy instances for a specific DC (As opposed to local dc only)

### CODE REVIEW

File: ob1k-consul/src/main/java/com/outbrain/ob1k/consul/ConsulHealth.java [hunk 1]
```diff
@@ -15,6 +15,8 @@
 
   ComposableFuture<List<HealthInfoInstance>> filterDcLocalHealthyInstances(final String service, final String filterTag);
 
+  ComposableFuture<List<HealthInfoInstance>> filterDcHealthyInstances(final String dc, final String service, final String filterTag);
```
Comment (marenzo) [inline ob1k-consul/src/main/java/com/outbrain/ob1k/consul/ConsulHealth.java]: there's importance to fields order when using path params - service should be first, then dc, else not gonna work.

**Comment by eranharel** [comment]
If you still need this code, I will merge.
But if you're going to move the check to the deploy scripts, please close this - I prefer to keep the API as "local" as possible.

**Comment by guymarom** [comment]
I don't need this for my feature anymore

**Comment by guymarom** [comment]
I'll probably go with the scripts as Michael suggested. I'll close...

**Pull request closed without merging**